packages/suite/src/storage/migrations/index.ts

```diff
@@ -566,4 +568,83 @@ export const migrate: OnUpgradeFunc<SuiteDBSchema> = async (
         // @ts-expect-error
         backendSettings.delete('trop');
     }
+
+    if (oldVersion < 37) {
+        await updateAll(transaction, 'accounts', account => {
+            // @ts-expect-error
+            if (!account.metadata?.fileName || !account.metadata?.aesKey) {
+                return account;
+            }
+            account.metadata = {
+                key: account.metadata.key,
+                1: {
+                    // @ts-expect-error
+                    fileName: account.metadata.fileName,
+                    // @ts-expect-error
+                    aesKey: account.metadata.aesKey,
+                },
+            };
+
+            return account;
+        });
+
+        await updateAll(transaction, 'devices', device => {
+            if (
+                device.metadata.status === 'enabled' &&
+                // @ts-expect-error
+                device.metadata.fileName &&
+                // @ts-expect-error
+                device.metadata.aesKey
+            ) {
+                device.metadata = {
+                    key: device.metadata.key,
+                    status: device.metadata.status,
+                    1: {
+                        // @ts-expect-error
+                        fileName: device.metadata.fileName,
+                        // @ts-expect-error
+                        aesKey: device.metadata.aesKey,
+                    },
+                };
+            }
+
+            return device;
+        });
+
+        await updateAll(transaction, 'metadata', metadata => {
+            const updatedMetadata = {
+                selectedProvider: { labels: '' },
+                providers: [],
+                enabled: metadata.enabled,
+            };
+            // @ts-expect-error
+            if (metadata.provider) {
+                // not sure about google client id I selected
```

Comment:
what do you think @komret which one should be selected if we detect that user has connect google provider?

Reply:
I don't fully understand the question actually, what's the problem with that ID? looks like it was there for a long time.

Reply:
there are two variants of clientId for google drive because there are two autorization flows for that provider, it can be recognized by refreshToken, only one of those flow use it